src/witness/observerAnalysis.ml

```diff
@@ -65,7 +65,10 @@ struct
     (* ctx.local doesn't matter here? *)
     [ctx.local, step ctx.local ctx.prev_node (FunctionEntry f)]
 
-  let combine ctx (lval:lval option) fexp (f:fundec) (args:exp list) fc (au:D.t) (f_ask: Queries.ask) : D.t =
+  let combine_env ctx lval fexp f args fc au f_ask =
+    ctx.local
```

Comment:
The observer analysis is consistent with the witness lifter such that both consider the function call edge to be fully completed after the assign has also taken place since the assign is part of that edge.